### ISSUE TITLE
Testing?

### ISSUE THREAD

**Issue by antholeole** [felangel/sealed_flutter_bloc_samples#2]
tried to look at the examples for tests - doesn't seem like this package mixes well with `bloc_test` or similar. This seems very difficult to test

**Comment by felangel** [comment]
Hi @antholeole 👋 
Thanks for opening an issue!

The `sealed_flutter_bloc` package should work well with `bloc_test`. Is there something in particular you were having trouble with?

**Comment by antholeole** [comment]
Thanks for getting back to me so fast! I've created an adaption on `very_good_core`: I wrote this code as quick as possible. All it does is switch to `sealed_bloc`, and added a state `nullable`. 

[What would go in this line to make the test pass?](https://github.com/antholeole/very-good-sealed-nullable/blob/c572a54df118d61c3e4c2b44b449967cd67ae0f9/test/counter/cubit/counter_cubit_test.dart#L24) I've tried a bunch of things but nothing even got close.
